tests/sentry/api/endpoints/test_group_attachments.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import absolute_import
+
+import six
+from six.moves.urllib.parse import urlencode
+
+from sentry.models import EventAttachment, File
+from sentry.testutils import APITestCase
+
+
+class GroupEventAttachmentsTest(APITestCase):
+    def create_attachment(self, type=None):
+        if type is None:
+            type = "event.attachment"
+
+        self.file = File.objects.create(name="hello.png", type=type)
+        self.file.putfile(six.BytesIO("File contents here"))
+
+        self.attachment = EventAttachment.objects.create(
+            event_id=self.event.event_id,
+            project_id=self.event.project_id,
+            group_id=self.group.id,
+            file=self.file,
+            name="hello.png",
+        )
+
+        return self.attachment
+
+    def path(self, types=None):
+        path = u"/api/0/issues/%s/attachments/" % (self.group.id,)
+
+        query = [("types", t) for t in types or ()]
+        if query:
+            path += "?" + urlencode(query)
+
+        return path
+
+    def test_basic(self):
```

Comment:
You should also include tests for files not being found and the account not having access to event attachments features.

Reply:
Thanks! Added feature gating to the endpoint now. A test for excluding files is in `test_filter`.